ansible_vault_rotate/vault/file_test.py

```diff
@@ -60,3 +62,20 @@ def test_vault_file(self):
             editor = VaultEditor(create_vault_lib("default", "test123"))
             content = editor.plaintext(f.name).decode("utf8")
             self.assertEqual(content, "i am\na multiline\nstring\n")
+
+    def test_multiple_vault(self):
+        with NamedTemporaryFile("r", delete=True) as f:
+            rekey_file(self.fixture_name("multiple-vault.yml"), "testprod", "testprod123", f.name)
+
+            self.assertLineCount(f, 17)
+            os.chdir("/tmp") # work around for ansible path resolve issues
```

Comment:
question: is this really required? On my end it works without

Reply:
I copied it from the other test cases. Not sure why it is needed